From e4fe5516a6f6dc49f59d6d35b4ce8fbaba2bfea3 Mon Sep 17 00:00:00 2001
From: BORDEU Felipe <felipe.bordeu@safrangroup.com>
Date: Mon, 16 Dec 2024 11:08:51 +0100
Subject: [PATCH] Add Option to deactivate the CPack directives

---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cd4a809d..73227540 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -341,4 +341,7 @@ include(doxygen)
 #####         Create Package
 #####
 ###############################################################################
-include(packages)
+SET (MMG5_PACKAGE TRUE CACHE BOOL "add CPack directives")
+IF (MMG5_PACKAGE)
+  include(packages)
+ENDIF ( MMG5_PACKAGE )
-- 
2.43.0.windows.1

